Docs/add helm chart install instructions (#127)
neicnordic/neic-sda: docs/guides/charts-overview.md

> # Sensitive Data Archive Helm Charts

Reviewer: ```suggestion
# Sensitive Data Archive (SDA) Helm Charts
```

Reviewer: The abbreviation `SDA` and the full name `Sensitive Data Archive` are used interchangeably throughout the text. I think we can add the abbreviation `(SDA)` at the beginning of the text and then use the short name `SDA` afterwards unless we need to use the full name by purpose.

## Charts overview

The `neicnordic` Helm repository contains the following charts (for configuration details click on the links below):

> - [sda-svc - SDA services](https://github.com/neicnordic/sensitive-data-archive/blob/main/charts/sda-svc/README.md)

Reviewer: Shall we capitalize all these items, as the 
```sda-mq - SDA Message Broker``` 
you used?

>   This chart deploys the service components needed to operate the Sensitive Data Archive solution. The charts may include additional service components that might be beneficial for administrative operations or extending the Sensitive Data Archive solutions to facilitate different use cases.

Reviewer: ```suggestion
  This chart deploys the essential service components needed to operate the Sensitive Data Archive (SDA) solution. It may include additional service components that might be beneficial for administrative operations or extending the SDA solutions to facilitate different use cases.
```

## Usage

[Helm](https://helm.sh) must be installed to use the charts.
Please refer to Helm's [documentation](https://helm.sh/docs/) to get started.

With Helm properly installed, add the `neicnordic` Helm repository as follows:

```sh
helm repo add neicnordic https://neicnordic.github.io/sensitive-data-archive
helm repo update
```

You can then run

```sh
helm search repo neicnordic
```

to see the available charts.

## Installing the Charts

To install a chart with the release name `my-release`:

```sh
helm install my-release neicnordic/<chart-name>
```

To configure a Helm chart with your own values, you can copy the default `values.yaml` file from the chart to your local directory and modify it as needed, or using helm:

```sh
helm show values neicnordic/<chart-name> > <values-filename>.yaml
```

Then, you can install the chart with the following command:

```sh
helm install my-release -f <values-filename>.yaml neicnordic/<chart-name>
```

Example:

```sh
helm show values neicnordic/sda-svc > my-values.yaml

> vi my-values.yaml

Reviewer: Maybe change this to e.g. 
```
# modify the file `my-values.yaml` with your own settings
```

helm install my-release neicnordic/sda-svc -f my-values.yaml
```

For quick reference to Helm's chart management capabilities see [here](https://helm.sh/docs/intro/cheatsheet/#chart-management).

## Uninstalling the Chart

> To uninstall/delete the `my-release` deployment:

Reviewer: ```suggestion
To uninstall the `my-release` deployment:
```
